assets/queries/azureResourceManager/trusted_microsoft_services_not_enabled/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy[result] {
+	doc := input.document[i]
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Storage/storageAccounts"
+
+	value.properties.networkAcls.defaultAction == "Deny"
```

Comment:
I think here should check if it is not `"Allow"`, since `networkAcls` is not a required field and, if is missing, should be an issue:
```suggestion
	value.properties.networkAcls.defaultAction != "Allow"
```

Reply:
You rock 🚀